lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -231,6 +349,25 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true, { ignore: ['consecutive-duplicates-with-different-values'] }],
+	fix: true,
+
```

Comment:
Can you add an example for accept?
i.e. `display: inline-block; display: inline;` should remain `display: inline-block; display: inline;` after fix